maker-bundle/src/Resources/skeleton/content-element/ContentElement.tpl.php

```diff
@@ -0,0 +1,27 @@
+<?= "<?php\n" ?>
+
+declare(strict_types=1);
+
+namespace App\Controller\ContentElement;
+
+use Contao\ContentModel;
+use Contao\CoreBundle\Controller\ContentElement\AbstractContentElementController;
+use Contao\CoreBundle\ServiceAnnotation\ContentElement;
+use Contao\Template;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+<?php if (PHP_VERSION_ID >= 80000): ?>
+#[AsContentElement("<?= $elementName ?>", "<?= $category ?>")]
```

Comment:
@aschempp Is this OK or would you prefer `#[AsContentElement("<?= $elementName ?>", category: "<?= $category ?>")]`?

Reply:
I would prefer with named argument just to make it same as the annotation

Reply:
Changed in cf1253ff9506af0e77d0e861296f86236eff5926.